package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "inspire-api",
+  "version": "0.1.0",
+  "description": "A micro service that looks up an insprire id for a given lng lat",
+  "scripts": {
+    "start": "nodemon ./server.js --exec babel-node",
```

Comment:
`start` should run the app in production mode. We use `watch` as the task for developers with all batteries included hot reloading